src/components/Header.astro

```diff
@@ -1,160 +1,204 @@
-    <ul
-      class="linkUl flex flex-col sm:flex-row sm:justify-between items-center gap-2 sm:gap-20 lg:gap-8 xl:gap-16"
+  <div class="relative">
+    <div
+      class="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none"
+    >
+      <svg
+        class="w-4 h-4 text-gray-500 dark:text-gray-400"
+        aria-hidden="true"
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 20 20"
+      >
+        <path
+          stroke="currentColor"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+          stroke-width="2"
+          d="m19 19-4-4m0-7A7 7 0 1 1 1 8a7 7 0 0 1 14 0Z"></path>
+      </svg>
+    </div>
+    <input
+      type="search"
+      id="default-search"
+      class="block w-full p-4 pl-10 text-sm text-gray-900 border border-gray-300 rounded-lg bg-gray-50 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+      placeholder="Search Orgs..."
+      required
+    />
+    <button
+      type="submit"
+      class="text-white absolute right-2.5 bottom-2.5 font-medium rounded-lg text-sm px-4 py-2 bg-highlight-blue hover:bg-dark-blue duration-200"
+      >Search</button
     >
-      <li>
-        <a
-          href="/"
-          class="animSlowOpacity font-subHeader hover:text-dark-blue duration-200 hover:text-3xl"
-          >Home</a
-        >
-      </li>
-      <li>
-        <a
-          href="https://google.com/"
-          class="animSlowOpacity slowOpacityDelay1 font-subHeader hover:text-dark-blue duration-200 hover:text-3xl"
-          >Organizations</a
-        >
-      </li>
-      <li>
-        <a
-          href="https://google.com/"
-          class="animSlowOpacity slowOpacityDelay2 font-subHeader hover:text-dark-blue duration-200 hover:text-3xl"
-          >Projects</a
-        >
-      </li>
-      <li>
-        <a
-          href="/team"
-          class="animSlowOpacity slowOpacityDelay3 font-subHeader hover:text-dark-blue duration-200 hover:text-3xl"
-          >Team</a
-        >
-      </li>
-    </ul>
   </div>
 </div>
-
-<style>
-  /* Typewriter */
-  .line1 {
-    border-right: 10px solid #0077cc;
-    text-align: center;
-    white-space: nowrap;
-    overflow: hidden;
-  }
-  .animTypewriter {
-    animation:
-      typewriter 1s steps(14) 1s 1 normal both,
-      blinkTextCursor 600ms steps(14) infinite normal;
-  }
-  @keyframes typewriter {
-    from {
-      width: 0;
-    }
-    to {
-      width: 9.1em;
-    }
-  }
-  @keyframes blinkTextCursor {
-    from {
-      border-right-color: #0077cce7;
-    }
-    to {
-      border-right-color: transparent;
-    }
+<nav>
+  {
+    navitems.map((item) => {
+      return <a href = {item.path}>{item.label}</a>;
+    })
   }
+</nav>
 
-  /* Link hover */
-  .linkUl li {
-    position: relative;
-    cursor: pointer;
-    transition: 0.5s;
-  }
+<style is:global>
```

Comment:
Many of the styles that you have added are are affecting the whole page. I would recommend shifting these styles to the `Layout.tsx` so that everyone is aware that these styles are globally available. Components should have only their own CSS, and not mess with the global CSS.